### PR TITLE
updated draggable component

### PR DESCRIPTION
**Pull request by GhaziCoorpacademy**
https://trello.com/c/OSG6VdWL/1212-rajouter-le-wrapper-drag-and-drop-sur-le-listitems-des-certification
**draggable published certifs (withSystemAll false):** 
image
**published certifs (withSystemAll true):** 
image

**Detailed purpose of the PR**

**Result and observation**

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Comment by GhaziCoorpacademy** [comment]
[rajouter le wrapper drag and drop sur le listItems des certification](https://trello.com/c/OSG6VdWL/1212-rajouter-le-wrapper-drag-and-drop-sur-le-listitems-des-certification)

**Review by silouone — COMMENTED** [review]
pk ya que des positions 1 ds ton screen ?

File: packages/@coorpacademy-components/src/molecule/draggable-list/test/fixtures/list-items.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const {
+  props: {items}
+} = listItems;
+
+export default {
+  props: {
+    itemType: 'list-item',
+    items,
+    onDrop: (dragged, dropped) => {
+      console.log('foo');
```
Comment (silouone) [inline packages/@coorpacademy-components/src/molecule/draggable-list/test/fixtures/list-items.js:12]: tu pexu clean celui la du coup

**Review by silouone — APPROVED** [review]
lgtm

File: packages/@coorpacademy-components/src/organism/list-item/index.js [hunk 2]
```diff
@@ -29,9 +29,11 @@ const ListItem = props => {
     <div className={style.wrapper}>
       {isPublished && contentType === 'certification' ? (
         <div className={style.orderWrapper}>
-          <div className={style.order} aria-label={ariaLabel}>
-            {order + 1}
-          </div>
+          {order !== null && order !== undefined ? (
```
Comment (silouone) [inline packages/@coorpacademy-components/src/organism/list-item/index.js:32]: necessaire ?

File: packages/@coorpacademy-components/src/molecule/draggable-list/index.js [hunk 3]
```diff
@@ -58,7 +58,7 @@ const DraggableList = ({items, onDrop, itemType}) => {
       onDragLeave={dragLeaveHandler}
       onDrop={dropHandler}
     >
-      <Item {...item} key={item.id} />
+      <Item {...item} key={item.id} order={index} />
```
Comment (silouone) [inline packages/@coorpacademy-components/src/molecule/draggable-list/index.js:61]: jpeferai gérer l'incrémentation ici que dans enfant ou on a pas le context

**Review by youssefezzahi96 — APPROVED** [review]

File: packages/@coorpacademy-components/src/organism/list-items/index.js [hunk 4]
```diff
@@ -1,19 +1,32 @@
...
-  const {items, itemType} = content;
-  const itemsView = items.map((item, index) => (
-    <li key={item.id} className={style.item} data-name={`content-${index}`}>
-      <ListItem {...item} order={index} contentType={itemType} />
-    </li>
-  ));
+  const {items, itemType, onDrop, isDraggable = false} = content;
+  const itemsView =
+    findIndex({type: 'published'}, items[0]?.tags) !== -1 &&
+    itemType === 'certification' &&
```
Comment (adamska27) [inline packages/@coorpacademy-components/src/organism/list-items/index.js:16]: on veut vraiment ça?

**Pull request merged**